### PR TITLE
creates dev help request issue template

### PR DESCRIPTION
**Pull request by BryanGK**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/dev-help-request-template.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+name: Developer Help Request Template
+about: Default developer help request template template
```
Comment (marcellmueller) [inline .github/ISSUE_TEMPLATE/dev-help-request-template.md:3]: @BryanGK 

Yeah I think this is a great idea :tada: Even just taking the time to clearly out what our issue is will help us to understand it better, and likely get better help. I guess my only question is the workflow for this is to open up and issue and get help/comments on it then close it?

Just noticed template written twice on this line as well as the spelling mistake in the commit title to cleanup.

Comment (BryanGK) [reply .github/ISSUE_TEMPLATE/dev-help-request-template.md]: Yeah we would probably need our own column in zenhub for these issues so they don't muddy the waters

Comment (BryanGK) [reply .github/ISSUE_TEMPLATE/dev-help-request-template.md]: Then close the issue once they get resolved

**Review by marcellmueller — APPROVED** [review]
Nice one! Minor request but can you squash this on merge because of that minor spelling error  :sweat_smile:

**Pull request merged**